### ISSUE TITLE
deosn't fit in terminal

### ISSUE THREAD

**Issue by nzjrs** [ionelmc/pytest-benchmark#30]
Since I upgraded to the latest beta, the pytest results now no longer fit in the terminal. The iterations column wraps around

**Comment by ionelmc** [comment]
Yeah ... it's a known issue. There are some plans to allow customizing the report in a `3.1.0` release (so it's more brief of whatever you like)

**Comment by ionelmc** [comment]
I'm afraid you'll have to use smaller fonts till `3.1` 😳

**Comment by nzjrs** [comment]
OK, so I guess I roll back to 2.X then. This is quite a usability regression.

**Comment by mje-nz** [comment]
Is this still going to be fixed? I've just started using pytest-benchmark and I'm impressed so far, but it's pretty hard to read the results.

**Comment by ionelmc** [comment]
Quick workaround: use `--benchmark-columns=min,max,mean,stddev,median`.

**Comment by mje-nz** [comment]
Great, thanks!

P.S. The documentation for --benchmark-columns has the default columns as 'min, max, mean, stddev, median, iqr, outliers, rounds, iterations' when it should be 'min, max, mean, stddev, median, iqr, outliers, **ops**, rounds, iterations'.
